features/org.wso2.carbon.appmgt.store.feature/src/main/resources/admin-dashboard/site/themes/default/templates/businessowners-new/js/businessowners-new.js

```diff
@@ -0,0 +1,172 @@
+/*
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+var Showalert = function(msg, type, target) {
+    type = type || 'info';
+    $('#'+ target)
+        .removeClass()
+        .addClass(type)
+        .addClass('alert')
+        .stop()
+        .fadeIn()
+        .delay(3000)
+        .fadeOut()
+        .find('#statusErrorSpan').html(msg);
+    var section=$('.title-section');
+    jQuery('html, body').animate({
+                                     scrollTop: section.offset().top
+                                 }, 1000);
+}
+
+
+var context = "/" + window.location.pathname.split("/")[1];  // default value is "/admin-dashboard"
+var fieldCount = 0;
+
+function completeAfter(cm, pred) {
```

Comment:
Move all these UI populating functions into a common .js file since the same functionality is repeated in many files